### PR TITLE
Sensitivity Index

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#357 
#### What does this implement/fix? Explain your changes.
- `sensitivity_index` method added

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri @alirezazolanvari

Hi folks ;-)

This PR is for #357 , please check carefully and give me some feedback ;-)
@alirezazolanvari please reformat added references (`76`) in `README.md` - `sensitivity_index` branch and then inform me.

**Notice 1** : Do not hurry, we have enough time ;-)
**Notice 2** : This PR has 2 reviewer, so please don't merge this PR to `dev` branch before finishing the discussion

Best Regards
Sepand Haghighi

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
Minor changes may be needed, I've commented them in review comments.

File: pycm/pycm_util.py [hunk 1]
```diff
@@ -517,3 +518,110 @@ def axes_gen(
...
+    :param mean: mean
+    :type mean: float
+    :param std: standard deviation
+    :type std: float
+    :return: normal distribution quantile as float
+    """
+    try:
+        return mean + std*math.sqrt(2)*inv_erf((2*p)-1)
+    except Exception:
+        return "None"
```
[the remark this answers is not in the document]
Comment (sepandhaghighi) [reply pycm/pycm_util.py]: fixed in 74ab6c4 and 997eb1f

File: pycm/pycm_util.py [hunk 2]
```diff
@@ -517,3 +518,110 @@ def axes_gen(
...
+def inv_erf(z):
+    """
+    Inverse error function.
+
+    :param z: function input
+    :type z: float
+    :return: result as float
+    """
+    if z < -1 or z > 1:
+        return "None"
```
[the remark this answers is not in the document]
Comment (sepandhaghighi) [reply pycm/pycm_util.py]: fixed in 7558048

**Review by sadrasabouri — APPROVED** [review]
Well done it looks good to me :+1:

File: Document/Document.ipynb [hunk 3]
```diff
@@ -12942,7 +13017,9 @@
     "\n",
     "<blockquote id=\"ref74\">74- A. F. Hayes and K. Krippendorff, \"Answering the call for a standard reliability measure for coding data,\" Communication methods and measures, vol. 1, no. 1, pp. 77-89, 2007.</blockquote>\n",
     "\n",
-    "<blockquote id=\"ref75\">75- M. Aickin, \"Maximum likelihood estimation of agreement in the constant predictive probability model, and its relation to Cohen's kappa,\" Biometrics, pp. 293-302, 1990.</blockquote>"
+    "<blockquote id=\"ref75\">75- M. Aickin, \"Maximum likelihood estimation of agreement in the constant predictive probability model, and its relation to Cohen's kappa,\" Biometrics, pp. 293-302, 1990.</blockquote>\n",
+    "\n",
+    "<blockquote id=\"ref76\">76- MacMillan, N.; Creelman, C. (2005). Detection Theory: A User's Guide. Lawrence Erlbaum Associates. ISBN 9781410611147.</blockquote>"
```
Comment (alirezazolanvari) [inline Document/Document.ipynb:13022]: Hi @sepandhaghighi 
Is this reference compatible in format with others?

Comment (sepandhaghighi) [reply Document/Document.ipynb:13022]: > Hi @sepandhaghighi
> Is this reference compatible in format with others?

@alirezazolanvari No and I asked you to reformat it in `README.md`, `sensitivity_index` branch

Comment (alirezazolanvari) [reply Document/Document.ipynb:13022]: Done!

Comment (sepandhaghighi) [reply Document/Document.ipynb:13022]: @alirezazolanvari
What's that `*` for?

**Comment by sepandhaghighi** [comment]
@alirezazolanvari 
Thanks for your effort 💯 
I updated `References` section in `Document.ipynb`  -->https://github.com/sepandhaghighi/pycm/pull/359/commits/d7b86c5b0e54e7fd32c2e77b9f2b50e6460a3b46
If everything is okay, please merge this PR after approval.

**Review by alirezazolanvari — APPROVED** [review]

**Pull request merged**